### ISSUE TITLE
Saving Changes is not working for configure settings even in 6.9

### ISSUE THREAD

**Issue by Avitale34** [edzillion/blood-n-guts#140]
As title states I am still having issues with the Configuration settings, save changes button. It works fine with Blood and Guts disabled.

**Comment by edzillion** [comment]
There were some changes that were accidentally left out of 0.6.9 - if you can test on 0.6.10?

**Comment by Avitale34** [comment]
Hmm still having same issue. It seems to trigger when I click on "Save Changes" I can close the settings with the Red X until I hit "Save Changes" After that it all starts acting funny I can't close it or hit save changes to do anything.

**Comment by Avitale34** [comment]
I also have 2 GMs in the game if that helps deduce anything.

**Comment by danheuck** [comment]
I'm having the same issue on 6.10

This is the error:
```
foundry.js:5258 SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at duplicate (foundry.js:979)
    at Object.onChange (settings.js:221)
    at ClientSettings._update (foundry.js:3500)
    at ClientSettings.set (foundry.js:3482)
    at SettingsConfig._updateObject (foundry.js:24922)
    at SettingsConfig._onSubmit (foundry.js:5256)
_onSubmit @ foundry.js:5258
async function (async)
_onSubmit @ foundry.js:5256
```
It looks like `canvas.scene.getFlag()` here sometimes returns undefined which causes the `duplicate()` call to error.

``` javascript
                //if the scenePool has increased in size we need to repopulate it
                const sceneSplats = duplicate(canvas.scene.getFlag(MODULE_ID, 'sceneSplats'));
                if (sceneSplats && sceneSplats.length) {
                    const trimmedSceneSplats = BloodNGuts.getTrimmedSceneSplats(sceneSplats);
                    // trim to new ScenePool size and draw
                    BloodNGuts.drawSceneSplats(trimmedSceneSplats);
                }
```

Let me know if you need any more information

**Comment by danheuck** [comment]
I went ahead and made a tiny pull request to fix this: #145

**Comment by edzillion** [comment]
Thanks @danheuck I have merged your PR and it will probably be included in a build later today.

**Comment by edzillion** [comment]
This is included in #150
